package.json

```diff
@@ -6,6 +6,10 @@
   "author": "Zain Fathoni <zain.fathoni@gmail.com>",
   "dependencies": {
     "@emotion/core": "10.0.10",
+    "@fortawesome/fontawesome-svg-core": "1.2.17",
```

Comment:
Please, make sure this actually works. 
an example, you can share a capture, like this https://github.com/devpendent/website/pull/3#discussion_r276883104

Reply:
yeah...i wonder the best method to test it. is a capture just okey?